commit e6eb5e3561ca8bcee04cfbe7de9acaa31159638e
Author: Jan Kiszka <jan.kiszka@siemens.com>
Date:   Thu Jan 24 08:52:33 2019 +0100

    arm64: dts: hikey: Give wifi some time after power-on
    
    commit 83b944174ad79825ae84a47af1a0354485b24602 upstream.
    
    Somewhere along recent changes to power control of the wl1835, power-on
    became very unreliable on the hikey, failing like this:
    
    wl1271_sdio: probe of mmc2:0001:1 failed with error -16
    wl1271_sdio: probe of mmc2:0001:2 failed with error -16
    
    After playing with some dt parameters and comparing to other users of
    this chip, it turned out we need some power-on delay to make things
    stable again. In contrast to those other users which define 200 ms, the
    hikey would already be happy with 1 ms. Still, we use the safer 10 ms,
    like on the Ultra96.
    
    Fixes: ea452678734e ("arm64: dts: hikey: Fix WiFi support")
    Cc: <stable@vger.kernel.org> #4.12+
    Signed-off-by: Jan Kiszka <jan.kiszka@siemens.com>
    Acked-by: Ulf Hansson <ulf.hansson@linaro.org>
    Signed-off-by: Wei Xu <xuwei5@hisilicon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/arch/arm64/boot/dts/hisilicon/hi6220-hikey.dts b/arch/arm64/boot/dts/hisilicon/hi6220-hikey.dts
index f4964bee6a1a..e47609aa20d6 100644
--- a/arch/arm64/boot/dts/hisilicon/hi6220-hikey.dts
+++ b/arch/arm64/boot/dts/hisilicon/hi6220-hikey.dts
@@ -118,6 +118,7 @@
 		reset-gpios = <&gpio0 5 GPIO_ACTIVE_LOW>;
 		clocks = <&pmic>;
 		clock-names = "ext_clock";
+		post-power-on-delay-ms = <10>;
 		power-off-delay-us = <10>;
 	};
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1127_e6eb5e3561ca8bcee04cfbe7de9acaa31159638e.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1127_e6eb5e3561ca8bcee04cfbe7de9acaa31159638e.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
